mlflow/ml-package-versions.yml

```diff
@@ -285,7 +285,6 @@ spark:
         git clone --depth 1 https://github.com/apache/spark.git $temp_dir
         git --git-dir=$temp_dir/.git rev-parse HEAD
         cd $temp_dir
-        export MAVEN_OPTS="-Xss256m -Xmx2g -XX:ReservedCodeCacheSize=1g"
```

Comment:
Removed this line to use `_COMPILE_JVM_OPTS` defined in `build/mvn` in the spark repository:

https://github.com/apache/spark/blob/79a6e00b7621bb3ebd99ff692452d292e11fb377/build/mvn#L32

```shell
_COMPILE_JVM_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=1g -Xss128m"
...
export MAVEN_OPTS=${MAVEN_OPTS:-"$_COMPILE_JVM_OPTS"}
```

Reply:
Not sure changing `-Xss256m` to `-Xss128m` can really fix the issue though.